Hashids/Hashids.swift

```diff
@@ -283,84 +256,71 @@ public class Hashids_<T: protocol<Equatable, UnsignedIntegerType>> : HashidsGene
-    return c;
+    return c
 }
 
-internal func unique<T:CollectionType where T.Generator.Element:Equatable>(a:T) -> [T.Generator.Element]
-{
+func unique<T: CollectionType where T.Generator.Element: Equatable>(a: T) -> [T.Generator.Element] {
     return transform(a, a) { (var c, a, b, e) in
-        if !c.contains(e)
-        {
-            c.append(e);
-        };
+        if !c.contains(e) {
+            c.append(e)
+        }
     }
 }
 
-internal func intersection<T:CollectionType where T.Generator.Element:Equatable>(a:T, _ b:T) -> [T.Generator.Element]
-{
+func intersection<T: CollectionType where T.Generator.Element: Equatable>(a: T, _ b: T) -> [T.Generator.Element] {
     return transform(a, b) { (var c, a, b, e) in
-        if b.contains(e)
-        {
-            c.append(e);
-        };
+        if b.contains(e) {
+            c.append(e)
+        }
     }
 }
 
-internal func difference<T:CollectionType where T.Generator.Element:Equatable>(a:T, _ b:T) -> [T.Generator.Element]
-{
+func difference<T: CollectionType where T.Generator.Element: Equatable>(a: T, _ b: T) -> [T.Generator.Element] {
     return transform(a, b) { (var c, a, b, e) in
-        if !b.contains(e)
-        {
-            c.append(e);
-        };
+        if !b.contains(e) {
+            c.append(e)
+        }
     }
 }
-internal func shuffle<T:MutableCollectionType, U:CollectionType where T.Index == Int, T.Generator.Element:UnsignedIntegerType, T.Generator.Element == U.Generator.Element, T.Index == U.Index>(inout source:T, salt:U)
-{
-    return shuffle(&source, salt: salt, saltRange: 0..<salt.count);
+func shuffle<T: MutableCollectionType, U: CollectionType where T.Index == Int, T.Generator.Element:UnsignedIntegerType, T.Generator.Element == U.Generator.Element, T.Index == U.Index>(inout source: T, salt: U) {
+    return shuffle(&source, salt: salt, saltRange: 0..<salt.count)
 }
 
-internal func shuffle<T:MutableCollectionType, U:CollectionType where T.Index == Int, T.Generator.Element:UnsignedIntegerType, T.Generator.Element == U.Generator.Element, T.Index == U.Index>(inout source:T, salt:U, saltRange:Range<Int>)
-{
-    let sidx0 = saltRange.startIndex, scnt = (saltRange.endIndex - saltRange.startIndex);
-    var sidx = source.count - 1, v = 0, _p = 0;
-    while(sidx > 0)
-    {
-        v = v % scnt;
-        let _i:Int = numericCast(salt[sidx0 + v]);
-        _p += _i;
-        let _j:Int = (_i + v + _p) % sidx;
-        let tmp = source[sidx];
-        source[sidx] = source[_j];
-        source[_j] = tmp;
-        v += 1;
-        sidx -= 1;
+func shuffle<T: MutableCollectionType, U:CollectionType where T.Index == Int, T.Generator.Element: UnsignedIntegerType, T.Generator.Element == U.Generator.Element, T.Index == U.Index>(inout source: T, salt: U, saltRange: Range<Int>) {
+    let sidx0 = saltRange.startIndex, scnt = (saltRange.endIndex - saltRange.startIndex)
+    var sidx = source.count - 1, v = 0, _p = 0
+    while(sidx > 0) {
+        v = v % scnt
```

Comment:
And here, do these letters stand for things I don't know because I'm not a math nerd? I'm okay with that as the answer, but if the author of this was just making up letter names like i j k etc. it's probably worth thinking up better names?